hollow/src/main/java/com/netflix/hollow/api/codegen/HollowConsumerJavaFileGenerator.java

```diff
@@ -82,4 +137,17 @@ private String createFullPackageName(String packageName, String subPackageName,
     private boolean isEmpty(String value) {
         return value == null || value.trim().isEmpty();
     }
+
+    private void appendImportFromBasePackage(StringBuilder builder, String leaf) {
+        builder.append("import ").append(packageName).append(".").append(leaf)
+            .append(";\n");
+    }
+
+    private void addToSetIfNotPrimitive(Set<String> set, String... schemaNames) {
+        for (String schemaName : schemaNames) {
+            if (!HollowCodeGenerationUtils.isPrimitiveType(schemaName)) {
```

Comment:
I initially checked `!config.isUseHollowPrimitiveTypes() || ` here, assuming that isUseHollowPrimitiveTypes being `false` meant that we were generating even primitive types, but the tests failed 😕